### PR TITLE
feat(install): configure installation from package.json

### PR DESCRIPTION
**Pull request by kjarmicki**
Continuation of #167 
I've tested it best to my ability, but it was a bit more complicated than previously due to multiple modules. Please check before merging.

### CODE REVIEW

File: packages/mongodb-memory-server-global-3.6/postinstall.js [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
   process.env.MONGOMS_DOWNLOAD_DIR = require('path').resolve(
     require('os').homedir(),
     '.cache',
     'mongodb-binaries'
   );
 }
-if (!process.env.MONGOMS_VERSION) {
+if (!resolveConfig('VERSION')) {
   process.env.MONGOMS_VERSION = '3.6.12'; // don't use `latest` it's nightly build
 }
```
Comment (nodkz) [inline packages/mongodb-memory-server-global-3.6/postinstall.js]: I think that better will be to create some method in `resolve-config` which setup default variables.

So 
```js
const resolveConfig = require('../mongodb-memory-server-core/lib/util/resolve-config').default;

if (!resolveConfig('VERSION')) {
  process.env.MONGOMS_VERSION = '3.6.12'; // don't use `latest` it's nightly build	
}	}
```

can be rewritten on something more clear:
```js
const setDefaultValue = require('../mongodb-memory-server-core/lib/util/resolve-config').setDefaultValue;

setDefaultValue('VERSION', '3.6.12'); // don't use `latest` it's nightly build	
```

`setDefaultValue` just for example, maybe you can suggest a better name.

File: README.md [hunk 2]
```diff
@@ -125,6 +126,22 @@ MONGOMS_MD5_CHECK=1 # if you want to make MD5 check of downloaded binary.
 # Passed constructor parameter `binary.checkMD5` has higher priority.
 ```
 
+```json
+"config": {
+  "mongoDbMemoryServer": {
```
Comment (nodkz) [inline README.md]: According to the naming of package `mongodb-memory-server` better will be to use the following key `mongodbMemoryServer` (lowercased D).

**Comment by nodkz** [comment]
Amazing! Thanks a lot 👍

**Comment by nodkz** [comment]
Published 5.1.0 on NPM 🚀

**Comment by kjarmicki** [comment]
Woohoo :) Thank you! :tada:

**Comment by BrandonLWhite** [comment]
Thanks @kjarmicki! I'm looking forward to trying this out!

**Comment by BrandonLWhite** [comment]
Can someone provide guidance on how I can configure the downloadDir to be %HOME/.cache from inside my package.json in a platform-agnostic way?

I basically want to achieve this:
`require('path').resolve(require('os').homedir(), '.cache', 'mongodb-binaries') ` but from package.json. 

Is this possible?

**Comment by nodkz** [comment]
@BrandonLWhite use another package https://www.npmjs.com/package/mongodb-memory-server-global

**Pull request merged**